cli/lsp/tsc.rs

```diff
@@ -4766,7 +4519,10 @@ mod tests {
     let result = request(
       &mut runtime,
       state_snapshot,
-      RequestMethod::Configure(ts_config),
+      TscRequest {
+        method: "$configure",
```

Comment:
How difficult would it be to change these tests to use `TsServer` and call the methods on it? This PR looks good, but ideally we'd be testing that struct at a higher level and not re-implementing implementation details in the tests.